contrib/drivers/huawei/oceanstor/client.go

```diff
@@ -992,16 +997,23 @@ func (c *OceanStorClient) RemoveLunGroupFromMappingView(viewId, lunGrpId string)
 		log.Infof("Lun group %s has already been removed from mapping view %s", lunGrpId, viewId)
 		return nil
 	}
+
 	url := "/mappingview/REMOVE_ASSOCIATE"
 	data := map[string]interface{}{
 		"ASSOCIATEOBJTYPE": ObjectTypeLunGroup,
 		"ASSOCIATEOBJID":   lunGrpId,
 		"TYPE":             ObjectTypeMappingView,
 		"ID":               viewId}
+
 	if err := c.request("PUT", url, data, nil); err != nil {
+		if c.checkErrorCode(err, ErrorLunGroupNotInMappingView) {
```

Comment:
Log the error

Reply:
I'd like to start another pr to add all these logs.